packages/ai-jsx/src/core/completion.tsx

```diff
@@ -189,7 +189,10 @@ export function CompletionProvider<T extends ModelPropsWithChildren>(
 }
 
 /** The default context used by {@link ChatProvider}. */
-const chatContext = AI.createContext<[ModelComponent<ModelPropsWithChildren>, ModelProps]>([AutomaticChatModel, {}]);
+export const chatContext = AI.createContext<[ModelComponent<ModelPropsWithChildren>, ModelProps]>([
```

Comment:
Why does this need to be exported?

Reply:
Good catch. It's a holdover from a previous iteration of this PR that's no longer necessary.